test/e2e/clusterclass_changes_test.go

```diff
@@ -54,7 +54,9 @@ var _ = Describe("When testing ClusterClass changes [ClusterClass]", func() {
 						"hostPath":      "/var/run/docker.sock",
 					},
 					map[string]interface{}{
-						"containerPath": "/tmp",
+						// /tmp cannot be used as containerPath as
+						// it already exists.
+						"containerPath": "/test",
```

Comment:
nit: maybe something more generic expandable like `/hostfs/tmp` would be better?

Reply:
not sure either :-)

Reply:
Let's play it safe :)